src/img/icons/ether.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="none" fill-rule="evenodd"><circle cx="16" cy="16" r="16" fill="#627EEA"/><g fill="#FFF" fill-rule="nonzero"><path fill-opacity=".602" d="M16.498 4v8.87l7.497 3.35z"/><path d="M16.498 4L9 16.22l7.498-3.35z"/><path fill-opacity=".602" d="M16.498 21.968v6.027L24 17.616z"/><path d="M16.498 27.995v-6.028L9 17.616z"/><path fill-opacity=".2" d="M16.498 20.573l7.497-4.353-7.497-3.348z"/><path fill-opacity=".602" d="M9 16.22l7.498 4.353v-7.701z"/></g></g></svg>
```

Comment:
The svg has been drawn for 32 x 32, i.e changing the view box causes clipping. I would need something to help me redraw it if I were to change the viewbox. Know of any tools?

Reply:
Now that I think about it, I believe this icon may not be needed.

I know the icon is in the design but in practice, we don't plug in token icons like this. The avatar should be generated based on the address like in the rest of the Dapp.

Reply:
@arrenv With respect to the Ether icon used in [the designs](https://www.figma.com/file/LlVmeMQxHVA04evfrgZyuN/Safe-Control?node-id=898%3A21233), should I leave in or use a blockie?

Reply:
It would be great to use a token icon where we can, and ETH I think at the very least is worth an icon. There is an existing one in the `img/tokens` folder, so, I would just use that.